contracts/ERC4626CashFlowLender.sol

```diff
@@ -284,35 +278,106 @@ contract ERC4626CashFlowLender is
     bytes32 policyData,
     bytes32 quoteSignatureR,
     bytes32 quoteSignatureVS,
-    uint40 quoteValidUntil
+    uint40 quoteValidUntil,
+    uint256 bucketId
   ) external onlyRole(POLICY_CREATOR_ROLE) returns (uint256 policyId) {
     uint256 balanceBefore = _balance();
-    policyId = riskModule().newPolicy(
-      payout,
-      premium,
-      lossProb,
-      expiration,
-      address(this),
-      policyData,
-      quoteSignatureR,
-      quoteSignatureVS,
-      quoteValidUntil
-    );
+    if (bucketId == type(uint256).max) {
+      policyId = SignedQuoteRiskModule(riskModule_).newPolicy(
+        payout,
+        premium,
+        lossProb,
+        expiration,
+        address(this),
+        policyData,
+        quoteSignatureR,
+        quoteSignatureVS,
+        quoteValidUntil
+      );
+    } else {
+      policyId = _createBucketPolicy(
+        riskModule_,
+        payout,
+        premium,
+        lossProb,
+        expiration,
+        address(this),
+        policyData,
+        quoteSignatureR,
+        quoteSignatureVS,
+        quoteValidUntil,
+        bucketId
+      );
+    }
     // Increases the debt
     _increaseDebt(balanceBefore - _balance());
     return policyId;
   }
 
   /**
-   * @dev Creates a new policy paid by this contract and increases the debt. See
-   * {SignedQuoteRiskModule.newPolicyPaidByHolder}
+   * @dev Creates several policies paid by this contract and increases the debt.
+   *
+   * Requirements:
+   * - Caller must have POLICY_CREATOR_ROLE
+   * - _balance() >= than the amount of the premium
```

Comment:
`   * - _balance() >= than the sum of the premiums`

Agregá a la función la documentación de los parámetros o al menos un comentario que indique que si es un RM sin bucket, hay que pasar type(uint256).max